addons/udes_stock/models/stock_picking_print_strategy.py

```diff
@@ -23,3 +23,14 @@ def strategies(self, picking):
         return self.search([
             ('picking_type_id', '=', picking.picking_type_id.id),
         ])
+
+    @api.multi
+    def records(self, picking, report=None):
```

Comment:
The `report` arg was added on the assumption that this would help a derived implementation of `records` select which records to print. The corresponding change in odoo-print isn't on master, which means that arg can be deleted.

Further, providing an explicit target could be made common in odoo-print. If a pair for, say, 'print_records' is passed in the context, then this could override any further selection.